package.json

```diff
@@ -61,6 +61,7 @@
     "gatsby-image": "2.2.38",
     "gatsby-plugin-emotion": "4.1.21",
     "gatsby-plugin-google-analytics": "2.1.33",
+    "gatsby-plugin-google-tagmanager": "^2.3.11",
```

Comment:
So for Flamelink App / API and Website we tend to keep the versions fixed and let dependabot help us update for security vulnerabilities etc. 
Can you please change this to be 
"gatsby-plugin-google-tagmanager": "2.3.11"

easiest it to uninstall gatsby-plugin-google-tagmanager and then yarn install gatsby-plugin-google-tagmanager@2.3.11

Reply:
sure thing. makes sense.